source/includes/_api_endpoint_campaigns.md

```diff
@@ -381,7 +381,7 @@ See JSON code example.
 <?php
 
 //...
-$response = $listApi->removeContact($contactId, $listId);
+$response = $listApi->removeContact($campaignId, $contactId);
```

Comment:
`$campaignId` should be `$segmentId`: https://github.com/mautic/api-library/blob/master/lib/Api/Segments.php#L77

Reply:
Oh, I just noticed it's in the campaigns endpoint. So there should be `$campaignApi` instead of `$listApi`. And ignore my previous comment. Can you update that?